datatypes/src/operations/reproject.rs

```diff
@@ -823,13 +824,14 @@ mod tests {
             input.unwrap(),
             SpatialPartition2D::new_unchecked((-180., 85.06).into(), (180., -85.06).into())
         );
-
-        assert_eq!(
-            output.unwrap(),
-            SpatialPartition2D::new_unchecked(
-                (-20_037_508.342_789_244, 20_048_966.104_014_594).into(),
-                (20_037_508.342_789_244, -20_048_966.104_014_594).into()
-            )
+        let output = output.unwrap();
```

Comment:
in general, it's not a bad idea to use approx equal here. Depending on the gdal/proj version the results may vary ever so slightly. But we should address that in a different PR.

Please revert this change and make sure you use GDAL 3.4.1 on your system instead. It's the default Gdal for ubuntu 22 LTS.

Reply:
0cfef8e